clients/cobol-lsp-vscode-extension/syntaxes/COBOL.tmLanguage.json

```diff
@@ -89,7 +80,117 @@
+      "patterns": [
+        {"include": "#cobol-condition-keyword"},
+        {"include": "#cobol-branch-keyword"},
+        {"include": "#cobol-callback-keyword"},
+        {"include": "#cobol-serialization-keyword"},
+        {"include": "#cobol-subroutine-keyword"},
+        {"include": "#cobol-exit-keyword"},
+        {"include": "#cobol-general-keyword"},
+        {
+          "match": "(?<![\\-\\w])(?i:END-EXEC)(?![\\-\\w])",
```

Comment:
This should be covered by the `end` block of `exec-cics-block` and `exec-sql-block`.

Reply:
updated